g11n-ws/vip-common/src/main/java/com/vmware/vip/common/i18n/status/APIResponseStatus.java

```diff
@@ -32,5 +32,7 @@ public class APIResponseStatus {
     public static final Response INTERNAL_NO_RESOURCE_ERROR = new Response(404,
             "Not Found");
     public static final Response UNKNOWN_ERROR = new Response(520, "Unknown Error");
-    public static final Response TRANSLATION_COLLECT_REQUEST_SUCCESS = new Response(200, "Source collection request have received");
+    public static final Response TRANSLATION_COLLECT_REQUEST_SUCCESS = new Response(200, "Source collection request have received(please use translation-product-component-api to confirm it)");
+    public static final Response TRANSLATION_COLLECT_SOURCE_PARAM_IS_FAlSE = new Response(605, "Check received the sources status, please use translation-product-component-api to confirm it.");
```

Comment:
What does 605 mean?

Besides I think you should point out "doesn't do source collection since 'collectSource' is false", contrary to TRANSLATION_COLLECT_REQUEST_SUCCESS.

Reply:
Different format response